README.md

```diff
@@ -103,6 +108,71 @@ $ export AWS_REGION=<YOUR_REGION>
 $ export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
 $ export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
 $ export AWS_CUSTOMER_MASTER_KEY_ID=<YOUR_CUSTOMER_MASTER_KEY_ID>
+$ export AWS_REGION=<YOUR_REGION>
+
+$ export AWS_ACCESS_KEY_ID_UPDATED=<YOUR_ACCESS_KEY_ID_UPDATED>
+$ export AWS_SECRET_ACCESS_KEY_UPDATED=<YOUR_SECRET_ACCESS_KEY_UPDATED>
+$ export AWS_CUSTOMER_MASTER_KEY_ID_UPDATED=<YOUR_CUSTOMER_MASTER_KEY_ID_UPDATED>
+$ export AWS_REGION_UPDATED=<YOUR_REGION_UPDATED>
+```
+~> **Notice:** For more information about the Encryption at Rest resource, see: https://docs.atlas.mongodb.com/reference/api/encryption-at-rest/
+
+- For `Private Endpoint Link` resource configuration:
+```sh
+$ export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+$ export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+$ export AWS_CUSTOMER_MASTER_KEY_ID=<YOUR_CUSTOMER_MASTER_KEY_ID>
+$ export AWS_REGION=<YOUR_REGION>
+$ export AWS_VPC_ID=<YOUR_VPC_ID>
+$ export AWS_SUBNET_ID=<YOUR_SUBNET_ID>
+$ export AWS_SECURITY_GROUP_ID=<YOUR_SECURITY_GROUP_ID>
+```
+~> **Notice:** For more information about the Encryption at Rest resource, see: https://docs.atlas.mongodb.com/reference/api/encryption-at-rest/
+
+
+##### AZURE env variables
+
+- For `Network Peering` resource configuration:
+```sh
+$ export AZURE_DIRECTORY_ID=<YOUR_DIRECTORY_ID>
+$ export AZURE_SUBCRIPTION_ID=<YOUR_SUBCRIPTION_ID>
+$ export AZURE_RESOURSE_GROUP_NAME=<YOUR_RESOURSE_GROUP_NAME>
+$ export AZURE_VNET_NAME=<YOUR_VNET_NAME>
+```
+~> **Notice:** For more information about the Network Peering resource, see: https://docs.atlas.mongodb.com/reference/api/vpc/
+
+
+- For Encryption at Rest resource configuration:
+```sh
+export AZURE_CLIENT_ID=<YOUR_CLIENT_ID>
+export AZURE_SUBCRIPTION_ID=<YOUR_SUBCRIPTION_ID>
+export AZURE_RESOURSE_GROUP_NAME=<YOUR_RESOURSE_GROUP_NAME>
+export AZURE_SECRET=<YOUR_SECRET>
+export AZURE_KEY_VAULT_NAME=<YOUR_KEY_VAULT_NAME>
+export AZURE_KEY_IDENTIFIER=<YOUR_KEY_IDENTIFIER>
+export AZURE_TENANT_ID=<YOUR_TENANT_ID>
+export AZURE_DIRECTORY_ID=<YOUR_DIRECTORY_ID>
+
+export AZURE_CLIENT_ID_UPDATED=<YOUR_CLIENT_ID_UPDATED>
+export AZURE_RESOURSE_GROUP_NAME_UPDATED=<YOUR_RESOURSE_GROUP_NAME_UPDATED>
+export AZURE_SECRET_UPDATED=<YOUR_SECRET_UPDATED>
+export AZURE_KEY_VAULT_NAME_UPDATED=<YOUR_KEY_VAULT_NAME_UPDATED>
+export AZURE_KEY_IDENTIFIER_UPDATED=<YOUR_KEY_IDENTIFIER_UPDATED>
+```
+~> **Notice:** For more information about the Encryption at Rest resource, see: https://docs.atlas.mongodb.com/reference/api/encryption-at-rest/
+
+##### GCP env variables
+- For `Network Peering` resource configuration:
+```sh
+$export GCP_PROJECT_ID=<YOUR_PROJECT_ID>
+```
+~> **Notice:** For more information about the Encryption at Rest resource, see: https://docs.atlas.mongodb.com/reference/api/encryption-at-rest/
```

Comment:
~> **Notice:** For more information about the Network Peering resource, see: https://docs.atlas.mongodb.com/reference/api/vpc/